### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: Startup error: get scenes or devices

### ISSUE THREAD

**Issue by marcodesantis71** [ilcato/homebridge-Fibaro-HC2#174]
Hi,

I've installed successfully the plugin in the past.

Today I receive this error on startup:

 [2019-12-2 14:06:23] [FibaroHC2] Error getting data from Home Center:  null
Dec  2 14:06:23 master homebridge_zwave[13646]: (node:13646) UnhandledPromiseRejectionWarning: Error: Startup error: get scenes or devices
Dec  2 14:06:23 master homebridge_zwave[13646]:     at /usr/lib/node_modules/homebridge-fibaro-hc2/dist/index.js:113:19
Dec  2 14:06:23 master homebridge_zwave[13646]:     at processTicksAndRejections (internal/process/task_queues.js:93:5)
Dec  2 14:06:23 master homebridge_zwave[13646]: (node:13646) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
Dec  2 14:06:23 master homebridge_zwave[13646]: (node:13646) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I try to reinstall the package, remove persist and accessory folders but nothing to do.

Any solution?

**Comment by ilcato** [comment]
Which version of nodejs do you have?

**Comment by marcodesantis71** [comment]
> Which version of nodejs do you have?

6.13.1 on ubuntu 16.04

**Comment by ilcato** [comment]
Can you upgrade to a more recent one?

**Comment by marcodesantis71** [comment]
> Can you upgrade to a more recent one?

Hi,

After HC2 restart the problem was solved.

Now I've another problem.

I create a new user on HC2. I create a variable (true/false) and create a scene with the new variable.
On config.json I set new variable on switchglobalvariables parameter.

On HC2 the new user has a permission for the scene.

When I try to start scene I receive this error:

There was a problem setting variable:  Gatto to true

If I use the admin user I have not a problem.
<img width="503" alt="Schermata 2019-12-03 alle 11 15 56" src="https://user-images.githubusercontent.com/23100741/70042023-7e96c100-15be-11ea-879a-e13912dea0ef.png">
<img width="730" alt="Schermata 2019-12-03 alle 11 16 24" src="https://user-images.githubusercontent.com/23100741/70042025-7e96c100-15be-11ea-9ae3-a16968b79f59.png">
<img width="731" alt="Schermata 2019-12-03 alle 11 16 37" src="https://user-images.githubusercontent.com/23100741/70042026-7f2f5780-15be-11ea-8896-c5a8b8ed4edb.png">
<img width="503" alt="Schermata 2019-12-03 alle 11 17 10" src="https://user-images.githubusercontent.com/23100741/70042027-7f2f5780-15be-11ea-8691-2096dfe19654.png">
